board/forms.py

```diff
@@ -3,15 +3,18 @@
 class PostForm(forms.Form):
     """
     A form representing the creation of a new post.
-    - `name` is the author's name
-    - `message` is the message being written in the post
-    - `photo` is the photo(s) attached to the post
 
-    None of the fields are required. However, at least either of the message or photo
+    The post may contain a name, message, or photo(s). None of the fields are required. 
+    However, at least either of the message or photo
     must exist for this to be a valid post.
+
+    Class Attributes
+        name -> string: the author's name
+        message -> string: the message being written in the post
+        photo -> ImageFiles: the photo(s) attached to the post    
```

Comment:
let's keep the type system consistent... Is this how we're going to be doing class attributes in this project?

Reply:
yeeeeeeeeeeeeeeeppppppppppppppppppppp